wdtk-client/src/main/java/org/wikidata/wdtk/client/DumpProcessingAction.java

```diff
@@ -101,4 +101,26 @@
 	 */
 	void setDumpInformation(String project, String dateStamp);
 
+	/**
+	 * Returns a report message containing information about the files which
+	 * were generated.
+	 * 
+	 * @return report message
+	 */
+	String getReport();
+
+	/**
+	 * Sets the actions name.
+	 * 
+	 * @param name
+	 */
+	void setActionName(String name);
+
+	/**
+	 * Returns a name which terms the action.
```

Comment:
Analogous to the setActionName documentation, this should read "Returns the name of the action.".